docs/_imke/clusterlifecycle/upgradingacluster/index.de.md

```diff
@@ -17,7 +17,7 @@ Infrastruktur sicher zu halten. In diesem Fall kann der nächste
 Block übersprungen werden. Nodes müssen dann nämlich dennoch
 selbst geupdated werden.
 
-Bevor Sie einen Cluster upgraden, lesen Sie bitte das [Changelog](/imke/about/kubernetesversions) der Ziel-Kubernetes-Version,
+Bevor Sie einen Cluster upgraden, lesen Sie bitte das [Changelog](/imke/about/changelog-v2.14.8/) der Ziel-Kubernetes-Version,
```

Comment:
Why use a distinct version here -> we need to update it on any new changelog?

Reply:
we don't have a global page

Reply:
then we should probably link to about/ if this is possible